### PR TITLE
[GPU] Fuse eltwise into MVN

### PR DESCRIPTION
**Pull request by jade-cho**
### Tickets:
 - *101692*

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp [hunk 1]
```diff
@@ -593,11 +593,11 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             return does_support_fusings;
         };
 
-        auto mvn_supports_fusings = [](mvn_node& node) -> bool {
+        auto mvn_supports_fusings = [](mvn_node& node, bool for_quantize = true) -> bool {
             auto in_layout = node.get_dependency(0).get_output_layout();
             if (node.get_primitive()->requires_alignment(in_layout.get_partial_shape()))
                 return false;
-            return data_type_traits::is_i8_u8(in_layout.data_type);
+            return for_quantize ? data_type_traits::is_i8_u8(in_layout.data_type) : true;
```
Comment (isanghao) [inline src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp:600]: what about: `data_type_traits::is_i8_u8(in_layout.data_type) || for_eltwise_fusion`

**Review by isanghao — CHANGES_REQUESTED** [review]

**Review by hyunback — APPROVED** [review]

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp (hunk 1 above)
Comment (jade-cho) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp]: fixed

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp [hunk 2]
```diff
@@ -593,11 +593,11 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
...
         };
```
[the remark this answers is not in the document]
Comment (jade-cho) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp:602]: Included

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp [hunk 3]
```diff
@@ -593,11 +593,11 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             return does_support_fusings;
         };
```
[the remark this answers is not in the document]
Comment (jade-cho) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp:594]: Tested.

**Comment by isanghao** [comment]
no perf regression: --this ~/log/pr/2023-05-16--21-20.report --ref ~/log/2023-05-16--17-00.report

**Review by isanghao — APPROVED** [review]

**Review by isanghao — APPROVED** [review]

**Pull request merged**